jerry-core/parser/js/opcodes-dumper.cpp

```diff
@@ -19,7 +19,7 @@
 #include "stack.h"
 #include "jsp-early-error.h"
 
-static idx_t temp_name, max_temp_name;
+static idx_t temp_name, max_temp_name, max_temp_name_for_local_var;
```

Comment:
It seems, that these names are not self-descriptive enough. Maybe it is better to name them:
`temp_reg, mar_temp_reg, max_temp_reg_for_local_var`?
And comments are needed.